pkg/reconciler/v1alpha1/taskrun/resources/pod.go

```diff
@@ -45,6 +45,11 @@ const workspaceDir = "/workspace"
 
 // These are effectively const, but Go doesn't have such an annotation.
 var (
+	groupVersionKind = schema.GroupVersionKind{
+		Group:   v1alpha1.SchemeGroupVersion.Group,
+		Version: v1alpha1.SchemeGroupVersion.Version,
+		Kind:    "TaskRun",
+	}
```

Comment:
is there some way to avoid duplicating this here and share it b/w this file + pkg/reconciler/v1alpha1/taskrun/taskrun.go ? (if it's worth the effort!)

Reply:
I just forgot to remove it in the other file :joy: